src/plugins/controls/public/control_group/editor/control_editor.tsx

```diff
@@ -129,7 +131,11 @@ export const ControlEditor = ({
       if (!mounted) return;
 
       const initialId =
-        embeddable?.getInput().dataViewId ?? getRelevantDataViewId?.() ?? (await getDefaultId());
+        dataViewId ??
```

Comment:
Can you explain what this change is for? Additionally, instead of passing it as a prop, it might be easier to just add the line `embeddable?.getOutput().dataViewIds?.[0]` here. That way you wouldn't need to rename the local variable below.